build.assets/build-package.sh

```diff
@@ -290,6 +290,10 @@ if [[ "${PACKAGE_TYPE}" != "pkg" ]]; then
         mv -v ${LINUX_CONFIG_FILE} ${PACKAGE_TEMPDIR}/buildroot${LINUX_CONFIG_DIR}
         CONFIG_FILE_STANZA="--config-files /src/buildroot${LINUX_CONFIG_DIR}/${LINUX_CONFIG_FILE} "
     fi
+
+    # include post-upgrade script
+    mv -v ${TAR_PATH}/examples/systemd/post-upgrade ${PACKAGE_TEMPDIR}
```

Comment:
I wish we moved production-worthy code outside of examples.

No action expected here, just chatting.

Reply:
Yeah, I'm a little reluctant to add this to the examples dir, but it seems like this is where the teleport systemd configuration lives.